index.html

```diff
@@ -1,300 +1,288 @@
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="images/linkedin.svg" alt="linkedin" />
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="images/medium.svg" alt="medium" />
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="images/github.svg" alt="github" />
-          </a>
-        </li>
-        <li>
-          <a href="#">
-            <img src="images/istock.svg" alt="istock" />
-          </a>
-        </li>
-      </ul>
-    </section>
-    <!-- Work Section -->
-    <section class="wrapper">
-      <div class="shot">
-        <div class="snapshot"><img src="images/Shot1.svg" alt="tonic" /></div>
-        <div class="block">
-          <h1 class="heading1">Tonic</h1>
-          <div class="subframe">
-            <p class="sub1">CANOPY</p>
-            <div class="circle"></div>
-            <p class="sub2">Back End Dev</p>
-            <div class="circle"></div>
-            <p class="sub2">2015</p>
-          </div>
-          <p class="normal">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+    <div>
+      <!--The nav bar-->
+      <nav>
+        <h3 class="logo indicator">My Logo</h3>
+        <div>
+          <ul id="desk-list">
+            <li class="indicator">Portoflio</li>
+            <li class="indicator">About</li>
+            <li class="indicator">Contact</li>
+          </ul>
+          <img
+            class="indicator"
+            id="union-menu"
+            src="images/Union.svg"
+            alt="Union"
+          />
+        </div>
+      </nav>
+      <!--The main section-->
+      <main>
+        <section>
+          <h1>I'm Kumilachew Glad to see you!</h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
           </p>
-          <ul class="icons">
-            <li class="iconsub">html</li>
-            <li class="iconsub">css</li>
-            <li class="iconsub">javaScript</li>
+          <a class="indicator btn-act">LET'S CONNECT</a>
+          <ul class="socials">
+            <li class="indicator">
+              <img src="./images/twitter.svg" alt="Twitter" />
+            </li>
+            <li class="indicator">
+              <img src="images/Linkedin.svg" alt="Linkedin" />
+            </li>
+            <li class="indicator">
+              <img src="images/medium.svg" alt="Medium" />
+            </li>
+            <li class="indicator">
+              <img src="images/github.svg" alt="Github" />
+            </li>
+            <li class="indicator">
+              <img src="images/angellist.svg" alt="Angellist" />
+            </li>
           </ul>
```

Comment:
You should increase the size of your icons and fix the LinkedIn icon issue 
![image](https://user-images.githubusercontent.com/43877689/177982700-b0950170-90c1-4e39-8911-3ad5b9c865a4.png)

Reply:
I have increased the size of the icon but no LinkedIn icon issue in my render. This look like this

![image](https://user-images.githubusercontent.com/65109017/177988884-def8d5be-08d9-4cbc-8b12-238c207d8176.png)

![image](https://user-images.githubusercontent.com/65109017/177989286-9fc343ae-702c-43d9-97ed-9ea75a511d5b.png)